src/main/kotlin/no/nav/arbeidsgiver/altinnrettigheter/proxy/altinn/AltinnClient.kt

```diff
@@ -55,7 +60,22 @@ class AltinnClient(val restTemplate: RestTemplate) {
                 throw RuntimeException(message)
             }
             respons.body!!
-        } catch (exception: RestClientException) {
+        } catch (exception: HttpClientErrorException) {
+            if (exception.statusCode.is4xxClientError) {
```

Comment:
Hvis vi bare sender videre 4xx, så vil ikke våre klienter f.eks. få "500 UserId: [none]"-feilen. Da må de inn i våre logger for å debugge, selv om feilen ligger hos dem. Jeg tror kanskje vi burde sende videre alle feil, uavhengig av statuskode